explorer/src/main/java/com/scalar/client/tool/explorer/Explorer.java

```diff
@@ -129,10 +127,14 @@ public JsonArray scan(String assetId, JsonObject condition) {
   }
 
   private JsonObject executeContract(String id, JsonObject argument) {
-    ClientServiceRequester requester =
-        () -> new ClientServiceResponse(clientService.executeContract(id, argument));
-    ClientServiceResponse response = fallback(requester);
-    return string2Json(response.getResult());
+    try {
+      ContractExecutionResult result = clientService.executeContract(id, argument);
+
+      return result.getResult().get();
+    } catch (ClientException clientException) {
+      fallback(clientException);
```

Comment:
It's more like a design question to the original creator (Craig...) but does it fallback in the first try every time (throwing the exception a client every time) ?

Reply:
@feeblefakie san
Yes, it happens only in the first try
Otherwise, we need to add initialize command

Reply:
The certificate and the contracts are always needed to be registered, I don't see any good reasons to register them in the exception handling. Is there any ? If not, can they be registered in the initialization time like in a constructor ?

Reply:
@feeblefakie 
Regarding certificate and contract registration via exception handling

If we added certificate and contract registration in the constructor in will try to register again and again. Certificates and contracts registered once do not need to register again.

I think the better method is to add one more command for certificate and contract registration like "get" command

could you please confirm which method will be adopted?

Reply:
How about adding another command like `initialize` ?

Reply:
@feeblefakie 
Can I proceed with the new command `initialize`

Reply:
@scalar-boney Do you see any concerns in the approach except for adding one more step for users ?
If not, let's go with it.

Reply:
@feeblefakie san
Thank you for the review comments.
Changes updated